model/src/main/kotlin/OrtResult.kt

```diff
@@ -385,6 +386,12 @@ data class OrtResult(
             !omitExcluded || !isExcluded(id)
         }
 
+    /**
+     * Return all [LicenseChoice]s for the [Package] with [id].
+     */
+    fun getLicenseChoices(id: Identifier): List<LicenseChoice> =
```

Comment:
I assumed that in future we'll have license choices also defined elsewhere, so the license choices do not only come from the OrtResult. This is why I looked up how path excludes and license finding curations are retrieved. 

I found that they get injected into `DefaultLicenseInfoProvider` to be used by the `LicenseInfoResolver`.
Have you considered processing license choices in a similar / analog way?

Reply:
Would it be ok for you to refactor this when other license choice configurations are added?

Reply:
I'd actually like to also have @mnonnenmacher opinion on that as he has written the parts relevant here before diving deeper into this. I fear that it could be quite a bit of tech debt, but let's see.

Reply:
I agree, the design idea is that the `LicenseInfoResolver` gets all required information from the `LicenseInfo` and therefore it would make sense to also add the choices there. But I would be fine with changing this separately, because I currently cannot decide which would be the best way to integrate the choices there.